pyproject.toml

```diff
@@ -26,6 +26,7 @@ dependencies = [
     "reframe-hpc >= 4.3.0, < 5.0.0",
     "matplotlib >= 3.0.0",
     "pandas >= 2.0.1",
+    "bokeh >= 3.2.0",
```

Comment:
Using latest bokeh version for now, but we can likely find an older version that works if needed.